editors/code/package.json

```diff
@@ -1,10 +1,11 @@
 {
-    "name": "ra-lsp",
-    "displayName": "ra-lsp",
+    "name": "rust-analyzer",
+    "displayName": "rust-analyzer",
```

Comment:
Maybe capital case this? Most extensions are capital cased

Reply:
Most, but not all, so I think it's better to stick to the "official" spelling of the project name: `rust-analyzer`. Should have chosen something single-worded and captializable instead, but too late to change now :)